srcpkgs/ghc-bin/template

```diff
@@ -31,6 +31,18 @@ ppc64le)
 	distfiles="https://alpha.de.repo.voidlinux.org/distfiles/ghc-${version}-powerpc64le-void-linux.tar.xz"
 	checksum=0c8df56aa0177459894ba53375115ab76f8eed9f4b1018fae5c332bcc6b79cb4
 	;;
+aarch64)
+	# create with "make binary-dist"
```

Comment:
I think the comment has the wrong indentation.

And yes, I think moving the manual bindists elsewhere would be best. Perhaps do like @q66 and ask to include in distfiles.voidlinux.org?